pkgs/servers/bazarr/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, lib, fetchurl, makeWrapper, python3, python3Packages, ... }:
+
+stdenv.mkDerivation rec {
+  pname = "bazarr";
+  version = "v0.8.4.3";
+
+  src = fetchurl {
+    url = "https://github.com/morpheus65535/bazarr/archive/${version}.tar.gz";
+    sha256 = "0vvx1lrngxq8c9i0ad8sy6sigcf67b2h7kkaxnlfdlnp1d8kmf1m";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    mkdir -p $out/src
+    cp -r * $out/src
+
+    mkdir -p $out/bin
+    makeWrapper "${python3.withPackages(ps: [ps.lxml])}/bin/python3" \
```

Comment:
Style: add a single space after withPackages, like this: `withPackages (ps:`.

You can also ask nix for the interpreter directly, instead of appending `/bin/python3`:

```
(python3.withPackages (ps: [ ps.lxml ])).interpreter
```